scripts/python/read_write_model.py

```diff
@@ -296,17 +453,28 @@ def rotmat2qvec(R):
     return qvec
 
 
+parser = argparse.ArgumentParser(description='Read and write COLMAP binary and text models')
```

Comment:
No need to run this code when importing the model. Otherwise these arguments will be injected in other scripts importing this module. Can you move it to the main function?